qdrant-landing/content/articles/qdrant-1.3.x.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Introducing Qdrant 1.3.0"
+short_description: "Check out what Qdrant 1.3 brings to vector search"
+description: "Check out what Qdrant 1.3 brings to vector search"
+social_preview_image: /articles_data/qdrant-1.3.x/social_preview.png
+small_preview_image: /articles_data/qdrant-1.3.x/icon.svg
+preview_dir: /articles_data/qdrant-1.3.x/preview
+weight: 1
+author: David Sertic
+author_link: 
+date: 
+draft: false
+keywords:
+  - vector search
+  - new features
+  - oversampling
+  - grouping lookup
+  - io_uring
+  - appendable mmap
+  - group requests
+---
+
+A brand-new [Qdrant 1.3.0 release](https://github.com/qdrant/qdrant/releases/tag/v1.3.0) comes packed with a plethora of new features, performance improvements and bux fixes:
+
+1. [Asynchronous I/O interface:](#asychronous-io-interface) Reduce overhead by managing I/O operations asynchronously, and by minimizing the need for context switches.
+2. [Oversampling for Quantization:](#oversampling-for-quantization) Improve the accuracy and performance of your queries while using Scalar or Product Quantization.
+3. [Grouping API lookup:](#grouping-api-lookup) Storage optimization method that lets you look for points in another collection using group ids.
+4. [Qdrant Web UI:](#qdrant-web-user-interface) A convenient dashboard to help you manage data stored in Qdrant.
+5. [Temp directory for Snapshots:](#temporary-directory-for-snapshots) Set a separate storage directory for temporary snapshots on a faster disk.
+6. [Other important changes](#important-changes)
+
+Your feedback is valuable to us, and are always tying to include some of your feature requests into our roadmap. Join [our Discord community](https://qdrant.to/discord) and help us build Qdrant!.
+
+## New features
+
+### Asychronous I/O interface
+
+Going forward, we will support the `io_uring` asychnronous interface for storage devices on Linux-based systems. Since its introduction, `io_uring` has been proven to speed up slow-disk deployments wherever the OS syscall overhead gets too high and the software becomes IO bound.
+
+<aside role="status">This experimental feature works on Linux kernels > 5.4 </aside>
+
+`io_uring` uses a ring buffer data structure to queue and manage I/O operations asynchronously, minimizing the need for context switches and reducing overhead. It supports features like submission and completion queues, event-driven I/O, and batch processing, allowing for efficient handling of I/O events and reducing the CPU overhead associated with traditional I/O.
```

Comment:
```suggestion
`io_uring` uses two ring buffers to queue and manage I/O operations asynchronously, avoiding costly context switches and reducing overhead. Unlike mmap, it frees the user threads to do computations instead of waiting for the kernel to complete.
```